src/behavior-property-observer.js

```diff
@@ -13,15 +16,36 @@ export class BehaviorPropertyObserver {
   * @param propertyName The name of the property.
   * @param selfSubscriber The callback function that notifies the object which defines the properties, if present.
   * @param initialValue The initial value of the property.
+  * @param coerce Instruction on how to convert value in setter
   */
-  constructor(taskQueue: TaskQueue, obj: Object, propertyName: string, selfSubscriber: Function, initialValue: any) {
+  constructor(taskQueue: TaskQueue, obj: Object, propertyName: string, selfSubscriber: Function, initialValue: any, coerce?: CoerceInstruction) {
     this.taskQueue = taskQueue;
     this.obj = obj;
     this.propertyName = propertyName;
     this.notqueued = true;
     this.publishing = false;
     this.selfSubscriber = selfSubscriber;
     this.currentValue = this.oldValue = initialValue;
+    if (coerce !== undefined) {
+      this.setCoerce(coerce);
+    }
+  }
+  setCoerce(coerce) {
+    let c;
+    switch (typeof coerce) {
+    case 'function':
+      c = coerce; break;
+    case 'string':
+      c = coerceFunctions[coerce]; break;
+    default: break;
+    }
+    if (c === undefined) {
+      LogManager
```

Comment:
could this be put in the `default:` switch case?  Then move the `this.coerce` assignment into the function and string cases, enabling removal of the `let c`?